src/Resources/config/selector.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
-        <service id="sonata.notification.erroneous_messages_selector" class="Sonata\NotificationBundle\Selector\ErroneousMessagesSelector">
+        <service id="sonata.notification.erroneous_messages_selector" class="Sonata\NotificationBundle\Selector\ErroneousMessagesSelector" public="true">
```

Comment:
If this service is not always required, I think we could declare it as an alias of the private service `Sonata\NotificationBundle\Selector\ErroneousMessagesSelector` in order to improve the container performance.

Reply:
Take a look at this thread @phansys https://github.com/sonata-project/SonataAdminBundle/pull/6210#discussion_r463934870

what do you mean by improving performance using a public alias?

Reply:
>Take a look at this thread @phansys sonata-project/SonataAdminBundle#6210 (comment)

No problem. My suggestion had the intention to keep the service private (its id really doesn't matter for what I was trying to achieve).
>what do you mean by improving performance using a public alias?

What I mean is, based on the [docs](https://symfony.com/doc/current/service_container/alias_private.html#marking-services-as-public-private), there seems to be an implicit optimization when a private service keeps private, using a public alias to access it from DIC if required.
I'm not sure if my supposal is right or not, these are the relevant paragraphs:
>Private services are special because they allow the container to optimize whether and how they are instantiated. This increases the container’s performance. It also gives you better errors: if you try to reference a non-existent service, you will get a clear error when you refresh any page, even if the problematic code would not have run on that page.

>However, if a service has been marked as private, you can still alias it (see below) to access this service (via the alias).

If I'm wrong about this, please forget about my comment.